homeassistant/components/seventeentrack/manifest.json

```
@@ -1,8 +1,10 @@
 {
   "domain": "seventeentrack",
   "name": "17TRACK",
-  "codeowners": [],
+  "codeowners": ["@shaiu"],
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/seventeentrack",
+  "integration_type": "hub",
```

```suggestion
  "integration_type": "service",
```
I'd argue that this is a service instead of a hub